pkg/client/unversioned/clientcmd/overrides.go

```diff
@@ -121,6 +138,7 @@ const (
 	FlagImpersonate  = "as"
 	FlagUsername     = "username"
 	FlagPassword     = "password"
+	FlagTimeout      = "request-timeout"
```

Comment:
@kubernetes/kubectl Any comments on the flag name?  `--timeout` was already take by commands that use reapers for a different meaning.